docs/components/connectors/protocol/soap.md

```diff
@@ -0,0 +1,212 @@
+---
+id: soap
+title: SOAP Connector
+sidebar_label: SOAP Connector
+description: The SOAP Connector allows you to connect your BPMN process with SOAP services.
+---
+
+:::note
+The **SOAP Connector** is only supported by Self-Managed Camunda 8 instances.
+:::
+
+**Simple Object Access Protocol (SOAP)** is a messaging protocol specification for exchanging structured
+information in the implementation of web services in computer networks.
+
+The **SOAP Connector** allows you to interact with [SOAP](https://en.wikipedia.org/wiki/SOAP) service endpoints
```

Comment:
Is this the best link we have for info on SOAP? IBM and Oracle have some docs on this too, but Wikipedia might be the best we can do for now.

Reply:
AFAIK, IBM and Oracle have their sort of "proprietary" flavors. If wikipedia is not reliable source here, maybe consider w3c https://www.w3.org/TR/soap/?

Reply:
I'm good with either external link. Since SOAP is a protocol, everyone wants their own flavor 🍦

Reply:
I added the `https://www.w3.org/TR/soap/` link. I believe this one is the most generic one